### DISCUSSION TITLE
Just posting some known Clones to tamper with.

### DISCUSSION THREAD

**Discussion by RanarTheWrathful** [Taureon/aps-plus-plus#102] [General]
I search through the sites for clones that match the animation and style of arras, their wiki or link will be provided, I case you guys get bored and want to create stuff from them:
- https://spaceblast.io |https://io-games.fandom.com/wiki/Spaceblast.io
- https://diep.io (DUH)
- Doblons, the site is down|https://doblonsio.fandom.com/wiki/Build
-  https://woomy.arras.cx (duh)|https://woomyarrasio.fandom.com/wiki/Woomy-arras.io_Wiki
- Obstar, site is down and no wiki, but the code of it is found here: https://github.com/bag-of-bug/Obstar 
- https://Tanksmith.io |https://tanksmith-io.fandom.com/wiki/Tanksmith.io_Wiki

**Comment by Taureon** [comment]
dublons.io is a parked page, also wrong domain
its doblons.io, like the wiki that you copied said
doesnt matter anyway, doblons is actually dead so why even inline-link it lol

**Reply by RanarTheWrathful** [reply]
I had made a mistake there, I didn't check the site and the mispel wasn't fixed, it should be edited now

**Comment by RanarTheWrathful** [comment]
+https://bloble.io | https://bloble.fandom.com/wiki/Bloble.io_Wiki
